mesonbuild/mtest.py

```diff
@@ -758,14 +758,14 @@ def print_stats(self, numlen: int, tests: typing.List['TestSerialisation'],
 
     def print_summary(self) -> None:
         msg = '''
-Ok:                 %4d
-Expected Fail:      %4d
-Fail:               %4d
-Unexpected Pass:    %4d
-Skipped:            %4d
-Timeout:            %4d
-''' % (self.success_count, self.expectedfail_count, self.fail_count,
-            self.unexpectedpass_count, self.skip_count, self.timeout_count)
+Ok:                 {0:4}
+Expected Fail:      {0:4}
+Fail:               {0:4}
+Unexpected Pass:    {0:4}
+Skipped:            {0:4}
+Timeout:            {0:4}
```

Comment:
This is wrong, replace all of the `{0:4}` with `{:4}`, otherwise all of these are going to be `success_count`

Reply:
Fixed format spacing.  The 0 was an accident.